Assets/MixedRealityToolkit/Definitions/BoundarySystem/BoundaryType.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.MixedReality.Toolkit.Boundary
+{
+    public enum BoundaryType
```

Comment:
Can you add doc comments to this enum and values?

Reply:
Added!